### ISSUE TITLE
Article reading page doesn't provide link back to main Shiori page

### ISSUE THREAD

**Issue by axelsimon** [go-shiori/shiori#579]
Hi!

## Data
- **Shiori version**: 1.5.4
- **Database Engine**:  sqlite
- **Operating system**:  NixOS 23.05 (pre)
- **CLI/Web interface/Web Extension**: Web Interface

## Describe the bug / actual behaviour
Clicking on an article opens a new tab to read the article. This tab contains no link back to the main Shiori interface.

## Expected behaviour
I expect to be able to go back to the main Shiori interface from an article served by Shiori.

## Notes
I get that the idea is to have a main Shiori page and each article open in a new tab. Not having a link back to the main Shiori interface reduces the risk of people ending up with multiple tabs with the main Shiori page open, but not everyone wants to open everything in new tabs  all the time :)

**Comment by fmartingr** [comment]
I just got used to the fact that the article opens in a new tab and just close it when I want to go back to the main Shiori interface. I keep thinking that maybe we should add the readable to the main javascript interface instead of serve that in the backend, but it's some discussion I want to have once I finish the new API (#497). I should sit down and elaborate a proper roadmap too.

**Comment by fmartingr** [comment]
> I think serve that in backend is a good idea, and I like to avoid crapy JavaScript as much as possible.

Yeah, both ways can coexist, but on my mind, if I'm using the JS interface, I'd prefer everything to load that way. Helps making a PWA for mobile devices easier to develop too. But again, just random thoughts on my head.
